source/includes/usage-examples/sample-app.rb

```diff
@@ -0,0 +1,18 @@
+require 'bundler/inline'
+gemfile do
+  source 'https://rubygems.org'
+  gem 'mongo'
+end
+
+uri = "<connection string URI>"
+
+Mongo::Client.new(uri) do |client|
+  # start-db-coll
```

Comment:
Q: should these start/end comments be here?

Reply:
Nope, good call